frontend/packages/console-dynamic-plugin-sdk/scripts/create-pkg.ts

```diff
@@ -0,0 +1,26 @@
+import * as fs from 'fs';
+import * as readPkg from 'read-pkg';
+
+const PKG_NAME = '@openshift-console/dynamic-plugin-sdk';
+const EXTENSIONS_EXPORT = 'lib/extensions/index.js';
+const WEBPACK_PLUGIN_EXPORTS = './lib/webpack/ConsoleRemotePlugin.js';
+
+const createPackageJson = () => {
+  const packageJson = readPkg.sync({ normalize: false });
+  packageJson.name = PKG_NAME;
+  delete packageJson.private;
+  packageJson.license = 'Apache-2.0';
+  packageJson.main = EXTENSIONS_EXPORT;
+  packageJson.exports = {
```

Comment:
FYI, TS does not yet support `exports` in `package.json`. But it will be useful once added. For now, consumer of `dynamic-plugin-sdk` has to add `paths` into his `tsconfig.json`.